models/activity.js

```diff
@@ -0,0 +1,19 @@
+let mongoose = require('mongoose');
+
+let activitySchema = mongoose.Schema({
+  actor: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User',
+    required: false
+  },
+  timestamp: {
+    type: Date,
+    required: true
+  },
+  referenceId: {
+    type: String,
```

Comment:
This likely has to be generic to account for a variation of activity types.

Given a Deck object the activity item can infer the type of message based on that model.
Similarly, a Blog object contains enough information to build a different format of activity message.